build.cake

```diff
@@ -134,6 +153,14 @@ Task("Pack")
     DeleteFiles(artifactsDir + "*symbols*");
 });
 
+void EnableGitLink() {
+    var pdbs = GetFiles($"./source/Halibut/bin/{configuration}/**/Halibut.pdb");
+    foreach(var pdb in pdbs)
+    {
+        GitLink3(pdb);

```

Comment:
There is a new setting in netcore, I think is `<DebugType>embedded</DebugType>` or something else! Maybe we can use that for netcore builds?

Reply:
Interesting, will check it out